tests/document/document-change-event.js

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
Create seperate directory `tests/document/document` and put `document.js` and `change.js` (or `event-change.js` or `change-event.js`) there?